requirements.txt

```diff
@@ -2,6 +2,7 @@ Bootstrap_Flask==2.2.0
 Flask==2.3.2
 flask_sqlalchemy==3.0.5
 Flask_WTF==1.2.0
+Marshmallow==0.15.0
```

Comment:
why so old? https://pypi.org/project/marshmallow/

Reply:
or maybe it's flask-marshmallow ? https://pypi.org/project/flask-marshmallow/
little risky to use library which doesn't have even first version :) WHy you have decided to use this library? I am just curious :)

Reply:
yes, it was supposed to be flask-marshmallow! 
answering your question, I figured that if flask version exists, it might work better with flask than normal version, by default :) it's my first time using this library so I don't have any opinions

Reply:
```adds additional features to marshmallow, including URL and Hyperlinks fields for HATEOAS-ready APIs``` ok so I challenge you to implement HATEOAS :D

Reply:
challenge accepted!